static/js/brand-store/SectionNav/SectionNav.test.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import SectionNav from "./SectionNav";
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useParams: jest.fn().mockReturnValue({ id: "test" }),
+}));
+
+test("active state is set on snaps when on snaps section", () => {
+  render(
+    <Router>
+      <SectionNav sectionName="snaps" />
+    </Router>
+  );
+
+  expect(screen.getByText("Snaps").getAttribute("aria-selected")).toEqual(
+    "true"
+  );
+});
+
+test("active state is not set on members or settings when on snaps section", () => {
+  render(
+    <Router>
+      <SectionNav sectionName="snaps" />
+    </Router>
+  );
+
+  expect(screen.getByText("Members").getAttribute("aria-selected")).toEqual(
+    "false"
+  );
+
+  expect(screen.getByText("Settings").getAttribute("aria-selected")).toEqual(
+    "false"
+  );
+});
+
+test("active state is set on snaps when on members section", () => {
+  render(
+    <Router>
+      <SectionNav sectionName="members" />
+    </Router>
+  );
+
+  expect(screen.getByText("Members").getAttribute("aria-selected")).toEqual(
+    "true"
+  );
```

Comment:
It's good to see more testing-library-style queries 👍 

This is fine as is, but in case of fairly generic strings such as this it might better to be more specific and use `getByRole`.

Also helps the reader of this test understand what you're testing exactly.

```suggestion
  expect(screen.getByRole('link', {name: "Members" }).getAttribute("aria-selected")).toEqual(
    "true"
  );
```